src/AppController.m

```diff
@@ -2496,9 +2496,6 @@ -(void)viewArticlePages:(id)sender inPreferredBrowser:(BOOL)usePreferredBrowser
             }
 		}
 		[self openURLs:urls inPreferredBrowser:usePreferredBrowser];
-		
-		if (!db.readOnly)
-            [articleController markReadByArray:articlesWithLinks readFlag:YES];
 	}
```

Comment:
I agree this needed to be changed.
But I think that if the user has enabled the "Mark current article read after a short delay" preference, she expects that the articles she has deliberately opened are marked read.
Removing these lines changes this behavior when multiple articles are selected and opened at once.